plugins/leemons-plugin-leebrary/frontend/src/components/AssetList.js

```diff
@@ -287,7 +287,7 @@ const AssetList = ({
         value: item.key,
         label: item.name,
         icon: (
-          <Box style={{ height: 16, marginBottom: 5 }}>
+          <Box style={{ height: 16, marginBottom: 5, color: '#636D7D' }}>
```

Comment:
There is no token for this color? maybe you can use the "sx={(theme) => ({ color: theme.some.token })}" props?

Reply:
problem solved. Thanks!